File: invert.js
const assert = require('assert');
let a, b, temp;

a = 999;
b = 555;
temp = a;
a = b;
b = temp;

try {
  assert.equal(555,a);
  assert.equal(999,b);
} catch (e) {
  console.log(e);
}

File: C/invert.c
#include<stdio.h>
#include<assert.h>
int main()
{
  int a,b,temp;
  a = 999;
  b = 555;
  printf("Variável a: %d\n",a);
  printf("Variável b: %d\n",b);
  temp = a;
  a = b;
  b = temp;
  printf("Variável a: %d\n",a);
  printf("Variável b: %d\n",b);
  
  assert(a==555);
  assert(b==999);
  return 0;
}